website/content/docs/deprecation/index.mdx

```diff
@@ -18,25 +18,27 @@ This announcement page is maintained and updated periodically to communicate imp
+| Centrify Auth Method                                        | v1.15                    | v1.17          | v1.17           | Use as an external plugin, but support will not be available.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |                                                                                                                                                                                                                                                             |
+| AWS secrets engine field change                             | v1.16                    | N/A            | N/A             | The `security_token` field returned for AssumeRole and FederationToken credentials is deprecated in favor of the current term `session_token`.                                                                                                                                                                                                                                                                                                                                                                                                                             | [AWS secrets engine API documentation](/vault/api-docs/secret/aws)                                                                                                                                                                                          |
+| Azure secrets password policy                               | N/A                      | v1.16          | N/A             | The `password_policy` field is deprecated because msgraph-sdk-go API doesn't let you provide a password, instead returns one to the client, making `password_policy` field unusable.                                                                                                                                                                                                                                                                                                                                                                                       | [MSGRAPH-SDK-GO APP API](https://learn.microsoft.com/en-us/graph/api/application-addpassword?view=graph-rest-1.0&tabs=http) [MSGRAPH-SDK-GO SP API](https://learn.microsoft.com/en-us/graph/api/serviceprincipal-addpassword?view=graph-rest-1.0&tabs=http) |
```

Comment:
I think it would be okay to remove "SDK-GO" from here and just reference MS graph API since it's not really a limitation of the Go SDK itself but the API.

Thanks for including the API links here 👍